### PR TITLE
VAKazakov  Exigius material report

### PR DESCRIPTION
**Pull request by VAKazakov**

### CODE REVIEW

File: canonn/materialReport.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        payload={}
+        payload["system"]=self.system
+        payload["body"]=  self.body
+        payload["latitude"]=  self.lat
+        payload["longitude"]=  self.lon
+        payload["category"]=self.entry["Category"]
+        payload["journalName"]=self.entry["Name"]
+        #payload["journalLocalised"]=unicode(self.entry.get(u"Name_Localised"))
+        payload["count"]=self.entry["Count"]
+        #payload["distanceFromMainStar"] = #TODO find method to calculate distance
```
Comment (derrickmehaffy) [inline canonn/materialReport.py]: only should apply to materials on bodies

File: canonn/materialReport.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        #payload["playMode"] = #TODO find method to see play mode
```
Comment (derrickmehaffy) [inline canonn/materialReport.py]: Removed in favor of factionState

File: canonn/materialReport.py [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+from urllib import quote_plus
+import sys
+import json
+from emitter import Emitter
+from debug import Debug
+from debug import debug,error
+
+'''
+{      
+  "system": "string",
```
Comment (derrickmehaffy) [inline canonn/materialReport.py]: Remove examples

File: canonn/materialReport.py (hunk 1 above)
Comment (VAKazakov) [reply canonn/materialReport.py]: This row disabled

**Comment by VAKazakov** [comment]
few errors encountered

**Pull request merged**